ledger/ledger-on-sql/src/main/scala/com/daml/ledger/on/sql/Database.scala

```diff
@@ -6,6 +6,9 @@ package com.daml.ledger.on.sql
 import java.sql.{Connection, SQLException}
 import java.util.concurrent.Executors
 
+import com.daml.concurrent.FutureOf._
```

Comment:
```suggestion
```

Add `-Xsource:2.13` to the scalacopts for ledger-on-sql instead.

Reply:
Only historical reasons, probably. The flag changes what will compile and what will not compile, so you are not 100% guaranteed to not run into trouble.

Reply:
Looks like we do. I'll just delete it.